Vagrantfile

```diff
@@ -34,7 +34,7 @@ Vagrant.configure("2") do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder "www", "/var/www/html"
+  config.vm.synced_folder "www", "/var/www/html", id: "vagrant-root", :owner => "www-data", :group => "www-data", mount_options: ["dmode=775,fmode=664"]
```

Comment:
better mounting permissions for uploading and moving files in the php (src/cms/api/*.php)